### PR TITLE
#281 

### PR DESCRIPTION
**Pull request by raj-uimatic**
Updates and new additions on #281 
Additional feature : disable already selected Attributes in the developer tab

### CODE REVIEW

File: lib/classes/class-settings.php [hunk 1]
```diff
@@ -127,6 +127,48 @@ public function __construct($args = false) {
           }
           $this->set('property_types', $ar);
         }
+        
+        /* Standard Attributes 
+         * added 28/07/2016 @raj
+         * 
+         */
+
+        $PDF_unique = array(
```
Comment (andypotanin) [inline lib/classes/class-settings.php:136]: @raj-uimatic I believe this is the list of standard attributes? If so, can we have this configuration come from a JSON file? We need to make it easily updatable.

Comment (raj-uimatic) [inline lib/classes/class-settings.php:136]: @andypotanin suggested place to keep this json ?

Comment (pemaxim) [reply lib/classes/class-settings.php:136]: @raj-uimatic I know, I suggested to use `/static/config/default.settings.json`. But, let's move it to another place.

We have `composer.json` in root directory. Please, move json from `standard_attributes.json` to `extra.schemas.standard_attributes` in composer.json.

The main reason is to have one place where we store all json settings. Also, you have to localize this data.

Use the following method for getting and localize standard attributes:

https://github.com/wp-property/wp-property/blob/feature-standard-attribute/vendor/libraries/usabilitydynamics/lib-wp-bootstrap/lib/classes/class-bootstrap.php#L140-L174

User Singleton for it:

`ud_get_wp_property()->get_schema( 'use.your.path.here' )`

For localization, see:

https://github.com/wp-property/wp-property/blob/feature-standard-attribute/vendor/libraries/usabilitydynamics/lib-wp-bootstrap/lib/classes/class-bootstrap.php#L176-L195

AND

https://github.com/wp-property/wp-property/blob/feature-standard-attribute/vendor/libraries/usabilitydynamics/lib-wp-bootstrap/lib/classes/class-bootstrap.php#L176-L195

Let me know if you have questions

Comment (raj-uimatic) [reply lib/classes/class-settings.php:136]: @pemaxim I moved the json data to composer.json
get_schema seems to be working fine.

question:
1) It seems you pasted same link twice in your comment. Did you mean to provide some other link ?
2) should I add localisations like  following link for the data coming from config.json ?
https://github.com/wp-property/wp-property/blob/feature-standard-attribute/lib/classes/class-bootstrap.php#L161-L167

Comment (Mashenka) [reply lib/classes/class-settings.php:136]: @raj-uimatic Max should be available from 5.09 I hope

**Pull request merged**